lib/Test/PostgreSQL.pm

```diff
@@ -389,16 +425,26 @@ method start() {
 # It could probably be made more sane.
 method _find_port_and_launch() {
   my $tries = 10;
-  my $port = $self->base_port;
+
   srand(); # Re-seed the RNG in case the caller forked the process
+
+  # There is a significant chance that there might be more than
+  # one Test::Postgresql test in flight so we better randomize
+  # the port before even trying to start the first time. Does
+  # no harm if there is no concurrent Postgres running; if there is
+  # then our process will start up a bit faster.
+  my $port = $self->base_port + int(rand(10)) + 1;
```

Comment:
This might muddy the matters a bit since this change is also related to the issue; that said, I'm pretty confident that the real cause for the problem was in Unix socket path being too long since that was what Postgres was complaining about in its log.

Reply:
Seems like a good plan though.